src/hera/workflows/v5/script.py

```diff
@@ -74,6 +74,7 @@ def _build_source(self) -> str:
             Final formatted script.
         """
         if callable(self.source):
+            self.command = self.command or ["python"]
```

Comment:
If the source is defined and command isn't, we now default to python if the source is a callable.

Reply:
Maybe we can make this command a global param? 🤔 It would go well with `image`